repos/tap/actions/files/local/removePendingFile.js

```diff
@@ -14,7 +14,6 @@ export const removePendingFile = (activeFile, screenId) => {
   // Update active file to not be modified
   activeFile &&
```

Comment:
Pretty sure we can remove this whole bit of code.
The `modified` key doesn't exist any more. So we should not be setting it here

Reply:
ur 100% right.
removed